staging/src/k8s.io/client-go/transport/token_source_test.go

```diff
@@ -145,7 +145,7 @@ func TestCachingTokenSourceRace(t *testing.T) {
 			go func() {
 				defer wg.Done()
 				if _, err := ts.Token(); err != nil {
-					t.Fatalf("err: %v", err)
+					t.Errorf("err: %v", err)
```

Comment:
`t.Fatalf` exits early but `t.Errorf` doesn't. You could pass the error to a channel and wait for the channel to receive or to be closed below, e.g.
```
go func() {
  wg.Wait()
  close(errc)
}
if err, ok := <-errc; ok {
  t.Fatalf("err: %v", err)
}
```

Reply:
@tiloso Thanks for your review! Please check again.

Reply:
Thanks a lot! That looks good to me.